templates/common/infra/bicep/core/host/container-app.bicep

```diff
@@ -28,6 +28,9 @@ param containerName string = 'main'
 @description('The name of the container registry')
 param containerRegistryName string = ''
 
+@description('Hostname suffix for container registry. Set when deploying to sovereign clouds')
+param containerRegistryHostSuffix string = 'azurecr.io'
```

Comment:
I'd recommend we split out all the `template` updates into a following PR after the bits are merged in.  I don't recall having a good experience validating template changes that require code changes.

Reply:
https://github.com/Azure/azure-dev/pull/3481